### PR TITLE
Add DevPullQueue to send TaskQueue requests to AppTaskQueue

### PR DESCRIPTION
**Pull request by menivaitsi**
https://ocd.appscale.com:8080/job/Daily%20Build/1778/

### CODE REVIEW

**Comment by cdonati** [comment]
:+1:

File: AppServer_Java/src/com/google/appengine/api/taskqueue/dev/UrlFetchJob.java [hunk 1]
```diff
@@ -168,17 +140,16 @@ private void addHeadersToFetchRequest( URLFetchServicePb.URLFetchRequest.Builder
...
-        if (queueXmlEntry.getTarget() != null)
-        {
+        if (queueXmlEntry.getTarget() != null) {
             requestProto.addHeader(buildHeader("X-AppEngine-ServerName", queueXmlEntry.getTarget()));
         }
         requestProto.addHeader(buildHeader("X-AppEngine-TaskExecutionCount", Integer.valueOf(retryCount).toString()));
 
-        if (previousResponse > 0) requestProto.addHeader(buildHeader("X-AppEngine-TaskPreviousResponse", Integer.valueOf(previousResponse).toString()));
+        if (previousResponse > 0)
+            requestProto.addHeader(buildHeader("X-AppEngine-TaskPreviousResponse", Integer.valueOf(previousResponse).toString()));
```
Comment (tmarballi) [inline AppServer_Java/src/com/google/appengine/api/taskqueue/dev/UrlFetchJob.java]: Please change the string to used the constant defined above.

File: AppServer_Java/src/com/google/appengine/api/taskqueue/dev/UrlFetchJob.java [hunk 2]
```diff
@@ -30,133 +29,106 @@
...
+public class UrlFetchJob implements Job {
+    private static final Logger logger = Logger.getLogger(UrlFetchJob.class.getName());
+    static final String X_GOOGLE_DEV_APPSERVER_SKIPADMINCHECK = "X-Google-DevAppserver-SkipAdminCheck";
+    static final String X_APPENGINE_QUEUE_NAME = "X-AppEngine-QueueName";
+    static final String X_APPENGINE_TASK_NAME = "X-AppEngine-TaskName";
+    static final String X_APPENGINE_TASK_RETRY_COUNT = "X-AppEngine-TaskRetryCount";
+    static final String X_APPENGINE_TASK_EXECUTION_COUNT = "X-AppEngine-TaskExecutionCount";
+    static final String X_APPENGINE_TASK_ETA = "X-AppEngine-TaskETA";
+    static final String X_APPENGINE_SERVER_NAME = "X-AppEngine-ServerName";
+    static final String X_APPENGINE_TASK_PREVIOUS_RESPONSE = "X-AppEngine-TaskPreviousResponse";
```
Comment (tmarballi) [inline AppServer_Java/src/com/google/appengine/api/taskqueue/dev/UrlFetchJob.java]: All these constants are defined but not used and the strings are used directly instead. Do you mind changing those?

**Comment by menivaitsi** [comment]
Build: https://ocd.appscale.com:8080/job/Create-Eucalyptus-Image/2646/

**Pull request merged**